runtime/vm/jvminit.c

```diff
@@ -54,6 +54,7 @@
 #include <signal.h>
 #endif
 
+#include "atoe.h"
```

Comment:
@Deigue We may need this guarded by platform - See failures on other platform (http://vmfarm.rtp.raleigh.ibm.com/job_output.php?id=89250736)

Reply:
Done